src/main/java/core/basesyntax/DataConverter.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+import java.util.List;
+
+public interface DataConverter {
+    public List<FruitTransaction> convertToTransaction(List<String> data);
```

Comment:
According to the Java Language Specification, it is not recommended to declare interface methods as `public` because they are `public` by default. You should remove the `public` modifier to follow Java best practices and clean up the code.